src/beamlime/offline/data_feeder.py

```diff
@@ -0,0 +1,52 @@
+import asyncio
+
+import numpy as np
+from colorama import Fore, Style
+from PIL import Image
+from PIL.ImageOps import flip
+
+from ..core.application import BeamlimeApplicationInterface
+from ..resources.images import load_icon_img
+from ..resources.images.generators import fake_2d_detector_img_generator
+
+
+class Fake2dDetectorImageFeeder(BeamlimeApplicationInterface):
+    detector_size = None
+    num_frame = None
```

Comment:
Again, why class attributes and not instance attributes?

Reply:
Initially I did it since the `_run` is `staticmethod` and it needed to check the validity of these fields
but now I removed that logic so I can make them as instance attributes...! Thanks